pilot/proxy/envoy/discovery.go

```diff
@@ -300,6 +309,32 @@ func NewDiscoveryService(ctl model.Controller, configCache model.ConfigStoreCach
 		container.ServeMux.HandleFunc("/debug/pprof/trace", pprof.Trace)
 	}
 	out.Register(container)
+
+	if len(o.WebhookEndpoint) > 0 {
+		out.webhookEndpoint = o.WebhookEndpoint
+		transport := &http.Transport{}
+
+		if strings.Index(o.WebhookEndpoint, "https") == 0 {
+			transport.TLSClientConfig = &tls.Config{
+				InsecureSkipVerify: true,
```

Comment:
keep CA verification

Reply:
why? if I have another pod with Istio CA maintained mTLS certs, its sure to cause an issue right? due to the fact that the cert is not signed by a known authority.

Reply:
this is disabling security and is going to cause problems when users expect "https" to be secure.

Reply:
I vote we drop https support entirely for this first version.

Verifying based on publicly issued certs is a non-starter for basically every cluster. Getting it working correctly with Istio Auth for mTLS can be an increment if we get feedback that people would like it. The recommended deployment is within the same pod, so plaintext is fine.

Reply:
dropped